Fi33Bu33.py

```diff
@@ -0,0 +1,14 @@
+for item in range(1, 101):
+    if item % 3 == 0 and item % 5 == 0:

```

Comment:
How to simplify conditional expression given the fact that `3` and `5` numbers are relatively prime? In other words, what is a Least Common Multiple (LCM) of relatively prime numbers?